recipe/meta.yaml

```diff
@@ -1,62 +1,73 @@
-{% set version = "2.3.8" %}
+{% set version = "5.6.1" %}
+{% set name = "pymc" %}
 
 package:
-  name: pymc
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pymc-devs/pymc/archive/v{{version}}.tar.gz
-  sha256: 200a28469d574a370fa9b55cd92f112f8ad09f71674bdae514437039e953689b
+  url: https://github.com/pymc-devs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3445f91136e44fe02a3e5029b2ececd262bc6623d4622c02cc47da21fa634be2
 
 build:
   number: 0
+  # skip because arviz is not supported for py38
+  skip: True  # [py<39]
+  script_env:
+    - LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
```

Comment:
`-bundle` seems to create problems (see logs `osx-arm64`)

Reply:
Right. Thanks for catching that.